City-of-Helsinki/helsinki-paatokset
# PP-0: Fix doubled page titles

Fix doubled page titles.

To test:
- Run `drush cim -y; drush cr`
- Check policymaker subpages and see that the titles aren't doubled / look OK otherwise.
- Click throught pages of different content types. Check that all titles look OK.
